src/ServiceProvider.php

```diff
@@ -58,6 +60,36 @@ function ($app) {
             }
         );
 
+        $this->app->extend(
+            'view.engine.resolver',
+            function (EngineResolver $resolver): EngineResolver {
+                $debugbarConfig = $this->app['config']->get('debugbar', []);
+                if (empty($debugbarConfig['enabled']) ||
```

Comment:
This check returns `false` if there is no `` environment variable defined and contradicts the default behaviour of the package (which is always enabled):

```php
// config/debugbar.php
<?php

return [

    /*
     |--------------------------------------------------------------------------
     | Debugbar Settings
     |--------------------------------------------------------------------------
     |
     | Debugbar is enabled by default, when debug is set to true in app.php.
     | You can override the value by setting enable to true or false instead of null.
     |
     | You can provide an array of URI's that must be ignored (eg. 'api/*')
     |
     */

    'enabled' => env('DEBUGBAR_ENABLED', null),
```